tests/UI/specs/EvolutionGraph_spec.js

```diff
@@ -197,7 +197,7 @@ describe("EvolutionGraph", function () {
         });
         await page.reload();
         await showDataTableFooter();
-        await page.click('.activatePeriodsSelection');
+        await page.click('.activatePeriodsSelection:last');
```

Comment:
```suggestion
        await (await page.jQuery('.activatePeriodsSelection:last')).click();
```

(or something like that)

Otherwise the UI tests will keep failing:

>           Evaluation failed: DOMException: Failed to execute 'querySelector' on 'Document': '.activatePeriodsSelection:last' is not a valid selector.

Or we could treat those "single module widget iframes" like the dashboard widgets and hide the duplicate controls. Adding `and not isWidget` to the `showTableActionsInHeader` template variable should do that well enough, and we can use the old selector again.

Reply:
I went with the `and is not isWidget` approach, let's see how that behaves.

Reply:
Hmm, that didn't make any difference.